### ISSUE TITLE
[1.4.x] Module get not installed with these errors

### ISSUE THREAD

**Issue by Portugao** [Guite/MostGenerator#740]
```
The upload directory "%s" does not exist and could not be created. Try to create it yourself and make sure that this folder is accessible via the web and writable by the webserver.

An error was encountered while creating the tables for the %s extension. Please enable the development mode by editing the /app/config/parameters.yml file (change the env variable to dev) in order to reveal the error details (or look into the log files at /app/logs/).

Initialization of MUMUImageModule failed!
```
The tables are there and the upload folder has permission 777.

**Comment by Guite** [comment]
Please tell me the path of the upload folders.

**Comment by Portugao** [comment]
Sorry. I was unprecise. I spoke of the folder userdata. The upload directory was created:
```
userdata/MUMUImageModule/pictures/imageupload
```
Should i try to put the module into mu/muimage-modules? I have it in MU/MUImageModule now.

**Comment by Guite** [comment]
Fixed in c036329c874145e20f23ade32b40e5f57f7f4901
Please recheck in about 30 minutes after all build jobs completed.

**Comment by Portugao** [comment]
This message is gone; get another -> open new issue.
